src/tools/types/toolsJob.ts

```diff
@@ -38,6 +38,7 @@ export interface RunningJob extends BaseJob<JobTypes> {
   status: Status.RUNNING | Status.QUEUED;
   remoteID: string;
   timeSubmitted: number;
+  progress?: number; // [0,100]
```

Comment:
100 _should_ be the finished status. In my latest commit I've added a bit of logic to set to 100% on FINISHED (even though it's not used in the UI) as I've realised in the IndexedDB data it might stay stuck at a lower value when finished and might be unexpected when debugging in the future